ui/src/app/pages/app/component.ts

```diff
@@ -64,6 +65,12 @@ export class AppComponent implements OnInit {
         };
         console.log('To override the API URLs, try:\n' +
           'setAllOfUsApiUrl(\'https://host.example.com:1234\')');
+
+        const lastActive = window.localStorage.getItem(INACTIVITY_CONFIG.LOCAL_STORAGE_KEY_LAST_ACTIVE);
+        if (lastActive == null || Date.now() - parseInt(lastActive) > environment.inactivityTimeoutInSeconds * 1000) {
```

Comment:
Is there anything to prevent them from being forced to logout on their first sign in? Will we set the local storage value during the account creation flow?